tests/framework/installer/cockroachdb.go

```diff
@@ -117,7 +117,7 @@ func (h *InstallHelper) UninstallCockroachDB(systemNamespace, namespace string)
 	err = h.waitForCustomResourceDeletion(namespace, crdCheckerFunc)
 	h.checkError(err, fmt.Sprintf("failed to wait for crd %s deletion", namespace))
 
-	_, err = h.k8shelper.DeleteResource("namespace", namespace)
+	_, err = h.k8shelper.DeleteResourceAndWait(false, "namespace", namespace)
```

Comment:
what about the `DeleteResource` call on line 124, and the operator deletion via `kubectlWithStdin` on line 128?  

from your investigation, is it only the removal of CRD instances and namespaces that are causing slow-downs?

do we understand why those particular types of resources are taking longer in 1.11 (where the slow-down may be)?

Reply:
Yes, what i observed in the logs was that only calls to delete the crd and namespace were causing the timeouts.  The behavior around deleting deployments or other resources didn't change since they already had the behavior in previous releases that they would wait for child resources (replicasets/pods) to be deleted before continuing.